### PR TITLE
Style/main page

### PR DESCRIPTION
**Pull request by Melizzo**
# PR Description
Please include a summary of the change:
 Add font for styling

Commit Emojis:
:tada: Initial commit
:art: CSS modifications/updates.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
Added alert for under 21 for future extention. 

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Tests (non-breaking change which adds testing)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by st0nebreaker** [comment]
Thanks Mel!

File: public/index.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@500&display=swap" rel="stylesheet">
```
Comment (asodemann18) [inline public/index.html:17]: nice custom font!

File: src/Login/Login.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const Login: React.SFC<LoginProps> = (props) => {
 	};
 
 	const denyUser = (event: any) => {
-		//display message for minors
+		alert('non-alcoholic drinks listing coming soon!')
```
Comment (asodemann18) [inline src/Login/Login.tsx:18]: solid message!

**Review by asodemann18 — APPROVED** [review]
Looks good!

**Pull request merged**